Libraries/LibGfx/Bitmap.cpp

```diff
@@ -321,17 +321,18 @@ void Bitmap::set_alpha_type_destructive(AlphaType alpha_type)
     if (alpha_type == m_alpha_type)
         return;
 
+    if (m_format == BitmapFormat::BGRx8888 || m_format == BitmapFormat::RGBx8888)
+        return;
```

Comment:
Maybe this should still set `m_alpha_type` to the requested value, technically we should always pass along the right Skia alpha type based on the format but better to be safe than sorry?

Tangent: maybe "opaque" would be a nice third alpha type to have here, Skia probably has the right idea.

Reply:
> Maybe this should still set m_alpha_type to the requested value, technically we should always pass along the right Skia alpha type based on the format but better to be safe than sorry?

My latest push now sets updates the alpha type before returning

> Tangent: maybe "opaque" would be a nice third alpha type to have here, Skia probably has the right idea.

I agree, if we're going to have `AlphaType::Opaque`, we might want to stop specifying  whether the bitmap has an alpha channel in `BitmapFormat`.